src/Database/Connectors/As400Connector.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace DreamFactory\Core\As400\Database\Connectors;
+
+use Illuminate\Database\Connectors\Connector;
+use Illuminate\Database\Connectors\ConnectorInterface;
+
+class As400Connector extends Connector implements ConnectorInterface
+{
+    /**
+     * Establish a database connection.
+     *
+     * @param  array $config
+     * @return \PDO
+     * @throws \Exception
+     */
+    public function connect(array $config)
+    {
+        $dsn = $this->getDsn($config);
+        $options = $this->getOptions($config);
+
+        return $this->createConnection($dsn, $config, $options);
+    }
+
+    /**
+     * Create a DSN string from a configuration.
+     *
+     * @param  array $config
+     * @return string
+     */
+    protected function getDsn(array $config)
+    {
+        $dsnParts = [
+            'odbc:DRIVER={IBM i ACCESS ODBC Driver}',
+            'SYSTEM=%s',
+            'DATABASE=%s',
+            'UID=%s',
+            'PWD=%s',
+            'NAM=0',
+            'DBQ=%s'
+        ];
+
+        $dsnConfig = [
+            $config['host'],
+            $config['database'],
+            $config['username'],
+            $config['password'],
+            $config['username'],
```

Comment:
@anas-srikou Do we need to validate `null` and/or empty checking for all `$config` params?

Reply:
@krishnapriawan Updated the code to deal with null value